Vagrantfile_nodes

```diff
@@ -53,10 +53,13 @@ $kubeMinionScript = <<SCRIPT
 
 set -x
 kubeadm reset
-kubeadm join \
-    #{KUBEADM_JOIN_FLAGS} \
-    --discovery-token-unsafe-skip-ca-verification \
-    --token "#{KUBETOKEN}" "#{MASTER_IP}:6443"
+
+if [ -n "#{KUBEADM_JOIN_FLAGS}" ]; then
+    kubeadm join #{KUBEADM_JOIN_FLAGS} --discovery-token-unsafe-skip-ca-verification --token "#{KUBETOKEN}" "#{MASTER_IP}:6443"
```

Comment:
Could you please as before split the flags on multiple lines.

Reply:
Done, each argument has its own line again now.